Loop/Managers/DeviceDataManager.swift

```diff
@@ -364,6 +370,38 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
         }
     }
 
+    /// NonMySentry Battery Calculation for Alkaline and Lithuim  #141
+    ///
+    /// - parameter currVoltage: Current Voltage Reading from Pump
+    ///
+    public var x22BatteryPercentRemaining : Double = -1
```

Comment:
Since we already have a battery % for x23 devices, let's track battery % as a single ivar.

Updating that ivar can trigger both battery change event (`AnalyticsManager.sharedManager.pumpBatteryWasReplaced()`) and notification (`NotificationManager.sendPumpBatteryLowNotification()`), and then the StatusTableViewController can check one place for battery percent, instead of checking both `dataManager.latestPumpStatusFromMySentry.batteryRemainingPercent` and `dataManager.x22BatteryPercentRemaining`